File: RepoTransBench/target_projects/Java/Go/jiaqi_jmxterm/public_tests/print_stream_command_output_public_test.go
package public_tests

import (
	"bytes"
	"testing"

	"github.com/stretchr/testify/assert"
)

type PrintStreamCommandOutput struct {
	out *bytes.Buffer
}

func (p *PrintStreamCommandOutput) Print(s string) {
	p.out.WriteString(s)
}
func (p *PrintStreamCommandOutput) Println(s string) {
	p.out.WriteString(s + "\n")
}
func (p *PrintStreamCommandOutput) Flush() {}

func TestOutputPrintsMessage(t *testing.T) {
	out := new(bytes.Buffer)
	po := &PrintStreamCommandOutput{out: out}
	testMessage := "PublicTestLine 42"
	po.Print(testMessage)
	po.Flush()
	result := out.String()
	assert.Contains(t, result, testMessage)
}

func TestOutputPrintsAnotherMessage(t *testing.T) {
	out := new(bytes.Buffer)
	po := &PrintStreamCommandOutput{out: out}
	testMessage := "AnotherUniquePublicTest"
	po.Println(testMessage)
	po.Flush()
	result := out.String()
	assert.Contains(t, result, testMessage)
}

File: RepoTransBench/source_projects/Java/jiaqi_jmxterm/src/test/java/org/cyclopsgroup/jmxterm/io/PrintStreamCommandOutputPublicTest.java
package org.cyclopsgroup.jmxterm.io;

import org.junit.Test;
import java.io.ByteArrayOutputStream;
import java.io.PrintStream;

import static org.junit.Assert.*;

public class PrintStreamCommandOutputPublicTest {

    @Test
    public void testOutputPrintsMessage() {
        ByteArrayOutputStream out = new ByteArrayOutputStream();
        PrintStream ps = new PrintStream(out);
        PrintStreamCommandOutput po = new PrintStreamCommandOutput(ps);
        String testMessage = "PublicTestLine 42";
        po.print(testMessage);
        po.flush();
        String result = out.toString();
        assertTrue("Expected output to contain our test message", result.contains(testMessage));
    }

    @Test
    public void testOutputPrintsAnotherMessage() {
        ByteArrayOutputStream out = new ByteArrayOutputStream();
        PrintStream ps = new PrintStream(out);
        PrintStreamCommandOutput po = new PrintStreamCommandOutput(ps);
        String testMessage = "AnotherUniquePublicTest";
        po.println(testMessage);
        po.flush();
        String result = out.toString();
        assertTrue("Output should contain unique printed value", result.contains(testMessage));
    }
}